/*
Die Funktion nimmt einen Error auf
 */
export const APIError = (error) => {
    alert(error);
};